### DISCUSSION TITLE
ESP32 Beta?

### DISCUSSION THREAD

**Discussion by SimonFili** [arendst/Tasmota#16322] [Q&A] [answered]
Hi there, reading from this page, many versions of ESP32 are still in BETA: https://tasmota.github.io/docs/ESP32/#esp32_1

If I'm reading this correctly, only ESP32-D0WD-V3 is not BETA?

Which version of hardware is out of BETA?  I want to build something reliable as possible.

Thanks!

**Comment by s-hadinger** [comment, marked as answer]
I guess we need to fix the documentation. Tasmota is known to be stable on all those esp32 variants

**Reply by SimonFili** [reply]
Ok, thanks for that.
